CONTRIBUTING.md

```diff
@@ -224,6 +264,9 @@ To run PostGraphile tests you will need to first create the
 createdb postgraphile_test
 ```
 
+The tests use the `TEST_PG_URL` environment variable as a connection URL, which can be overwritten.
```

Comment:
```suggestion
The PostGraphile tests use the `TEST_PG_URL` environment variable as a connection URL, which can be overwritten. (This differs from the Graphile Engine `TEST_DATABASE_URL` to avoid conflicts between these two independent test suites.)
```